docs/application/web/get-started/addon/first-addon.md

```diff
@@ -0,0 +1,520 @@
+
+To do that, right-click the project in the **Project Explorer** view and select **Build Signed Project**.
+
+**Figure: Building the addon**
+
+![Building the addon](media/build_manual_a.png)
+
+After you have built the addon, run it.
+
+<a name="run"></a>
+## Running your addon
+
+You can run the addon on the [emulator](../../tutorials/process/run-debug-app.md#emulator) or a [real target device](../../tutorials/process/run-debug-app.md#target).
+
+<a name="emulator"></a>
+### Running on emulator
+
+To run the addon on the emulator:
+
+1.  Launch an emulator instance in the [Emulator Manager](../../../tizen-studio/common-tools/emulator-manager.md):
+    1. In the Tizen Studio menu, select **Tools \> Emulator Manager**.
+
+        ![Emulator Manager](media/emulator_icon.png)
+
+    2.  In the Emulator Manager, select a mobile emulator from the list and click **Launch**.
+
+        If no applicable emulator instance exists, [create a new one](../../../tizen-studio/common-tools/emulator-manager.md#create).
+
+        ![Launching the emulator](media/emulator_launch_m.png)
+
+        The emulator is launched in its own window. You can also see the new emulator instance and its folder structure in the **Device Manager**.
+
+        ![Emulator](media/emulator_window_m.png)
+
+2.  Generate a security profile.
+
+    Before you run the addon, you must [sign your addon package with a certificate profile](../../../tizen-studio/common-tools/certificate-registration.md) in the Tizen Studio.
+
+3.  Run the addon:
+    
+    As mentioned, addons run as part of Web applications. Therefore, you cannot run them independently.
+
+    1. In the **Project Explorer** view, right-click the project and select **Run As \> Tizen Web Application**.
+
+        ![Running the application](media/app_run_w.png)
+
+        Alternatively, you can also select the project in the **Project Explorer** view and do one of the following:
+
+        -   Press the **Ctrl + F11** key.
+        -   Click the run icon in the toolbar.
+
+        If you have created multiple emulator instances, select the instance you want from the combo box in the toolbar before selecting to run the application. If you select an offline emulator, it is automatically launched when you select to run the application.
+
+        ![Selecting the emulator to use](media/app_run_multiple_emulators_m.png)
+
+    2.  In the error dialog that appears, to launch the application on the emulator, click **OK**.
+
+        ![Unable to run the addon](media/unable_to_run_addon.png)
+
+        The **WebApp Addon Setting** icon appears in the emulator.
+
+        ![Addon settings app](media/addon_settings_icon.png)
+
+        You can see that the addon is installed named as `tmplDefault` when you run the application.
+
+        ![Addon settings main](media/addon_settings_main.png)
+
+    3.  Now, you can run BasicUI Web application by following the [instruction](../mobile/first-app.md#emulator)
```

Comment:
done to add 'the' as suggested.

I want to keep the name as "BasicUI" because it's a proper noun of sample Web application
you may see in the screenshot (generate_hello_html.png) in the line 298 of this page

Reply:
Thank you for sharing your suggestion. If it is the UI name, then we cannot change.
However, in this file, many other places "Basic UI" is used instead of "BasicUI".
Line 34, 35, 37, 284, 286, 288, 409, and so on.
Please verify.

Reply:
OK. Please let me unify them as 'Basic UI', neither 'BasicUI' not 'basic UI'.